integration_tests/tests/assert_no_drop_in_data_returns_0_rows.sql

```diff
@@ -0,0 +1,10 @@
+select *
```

Comment:
Should we be more specific here?

Reply:
Yeh, probably doesn't matter.

Reply:
doesnt matter, we want to select everything and make sure we dont get any rows